index.js

```diff
@@ -14,16 +14,14 @@
  * limitations under the License.
  */
 
+const { setUnderTest } = require('./lib/utils/utils');
+setUnderTest(); // Note: we must call setUnderTest before initializing.
```

Comment:
Let's rename it to something like `setDevSetup` as it's also going to affect scripts importing local pw checkout via require('.')

Reply:
Done.